.github/scripts/generate_release_notes.py

```diff
@@ -144,6 +144,9 @@ def get_repo_priority(name):
 
 # generate changes and add contributors to set with or without release notes.
 for issueOrPR in issuesOrPRs:
+    if 'stale' in issueOrPR.labels:
```

Comment:
I am actually not sure if we should do this. An issue could be marked as stale but not closed by the Dapr Bot and worked as part of the milestone.

Reply:
Why need to consider it? We only put merged PRs into release notes, right? If so, they are not marked stale I think, since Bot would remove this label if active.

Reply:
This is about issues. We only put PRs in the milestone if we don't have an issue for it. Like I said, I think this PR should not be merged because it can accidentally include legit issues in the milestone that were marked with Stale label.